docs/reference/setup/logging-config.asciidoc

```diff
@@ -23,40 +23,42 @@ will resolve to `/var/log/elasticsearch/production.log`.
 appender.rolling.type = RollingFile <1>
 appender.rolling.name = rolling
 appender.rolling.fileName = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}.log <2>
-appender.rolling.layout.type = PatternLayout
-appender.rolling.layout.pattern = [%d{ISO8601}][%-5p][%-25c{1.}] [%node_name]%marker %.-10000m%n
-appender.rolling.filePattern = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}-%d{yyyy-MM-dd}-%i.log.gz <3>
+appender.rolling.layout.type = ESJsonLayout <3>
+appender.rolling.layout.type_name = server <4>
+appender.rolling.filePattern = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}-%d{yyyy-MM-dd}-%i.log.gz <5>
 appender.rolling.policies.type = Policies
-appender.rolling.policies.time.type = TimeBasedTriggeringPolicy <4>
-appender.rolling.policies.time.interval = 1 <5>
-appender.rolling.policies.time.modulate = true <6>
-appender.rolling.policies.size.type = SizeBasedTriggeringPolicy <7>
-appender.rolling.policies.size.size = 256MB <8>
+appender.rolling.policies.time.type = TimeBasedTriggeringPolicy <6>
+appender.rolling.policies.time.interval = 1 <7>
+appender.rolling.policies.time.modulate = true <8>
+appender.rolling.policies.size.type = SizeBasedTriggeringPolicy <9>
+appender.rolling.policies.size.size = 256MB <10>
 appender.rolling.strategy.type = DefaultRolloverStrategy
 appender.rolling.strategy.fileIndex = nomax
-appender.rolling.strategy.action.type = Delete <9>
+appender.rolling.strategy.action.type = Delete <11>
 appender.rolling.strategy.action.basepath = ${sys:es.logs.base_path}
-appender.rolling.strategy.action.condition.type = IfFileName <10>
-appender.rolling.strategy.action.condition.glob = ${sys:es.logs.cluster_name}-* <11>
-appender.rolling.strategy.action.condition.nested_condition.type = IfAccumulatedFileSize <12>
-appender.rolling.strategy.action.condition.nested_condition.exceeds = 2GB <13>
+appender.rolling.strategy.action.condition.type = IfFileName <12>
+appender.rolling.strategy.action.condition.glob = ${sys:es.logs.cluster_name}-* <13>
+appender.rolling.strategy.action.condition.nested_condition.type = IfAccumulatedFileSize <14>
+appender.rolling.strategy.action.condition.nested_condition.exceeds = 2GB <15>
 --------------------------------------------------
 
 <1> Configure the `RollingFile` appender
 <2> Log to `/var/log/elasticsearch/production.log`
-<3> Roll logs to `/var/log/elasticsearch/production-yyyy-MM-dd-i.log`; logs
+<3> Use Json layout.
+<4> `type_name` is a flag populating `type` field in a `ESJsonLayout`. It is used to distinguish streams of logs when parsing.
```

Comment:
Nit: "populating *the* `type` field". 

How about "It is used to distinguish streams of logs when parsing." -> "It can be used to distinguish different types of logs more easily when parsing them."?